src/components/shared/Validations.tsx

```diff
@@ -131,13 +131,13 @@ const NumberValidations = ({
     return null;
   }
   return (
-    <Flex color="muted" maxW="full">
+    <HStack color="muted" maxW="full" spacing={1}>
       {entries
         .map(([key, value]) => numberValidationFormatters[key](value))
         .map((value, i) => (
```

Comment:
feels redundant, but since it's an existing code and this PR is already fairly large, let's keep it unchanged